### ISSUE TITLE
our `imread` and `imwrite` are barely tested

### ISSUE THREAD

**Issue by hermancollin** [axondeepseg/axondeepseg#806]
**Describe the problem**
The `ads_utils.imread()` and `imwrite()` functions are barely unit tested. In _test_ads_utils.py_, we only have 2 tests on imread.

**Proposed solution**
Given how extensively we use these functions everywhere, I think we should add a few more unit tests. This is even more important now that ivadomed is no longer used, because we are responsible for more image validation and preprocessing on the ADS side.

**Comment by mathieuboudreau** [comment]
Yeah, I think we had more tests when we did the handling of everything in v3, but probably reloved them when we integrated to v4/ivadomed. Do tou have a list of checks tou think would be relevant specifically for nnunet use?
